source/type/pb_converter.cpp

```diff
@@ -750,6 +750,12 @@ void PbCreator::Filler::fill_pb_object(const nt::Network* n, pbnavitia::Network*
 void PbCreator::Filler::fill_pb_object(const nt::PhysicalMode* m, pbnavitia::PhysicalMode* physical_mode) {
     physical_mode->set_name(m->name);
     physical_mode->set_uri(m->uri);
+
+    if (depth >= 1 && m->co2_emission) {
+        pbnavitia::Co2EmissionRate* pb_co2_emission_rate = physical_mode->mutable_co2_emission_rate();
+        pb_co2_emission_rate->set_unit("gEC");
```

Comment:
```suggestion
        pb_co2_emission_rate->set_unit("gEC/km");
```
If I understood the ntfs doc correctly